tests/test_daemon.cpp

```diff
@@ -227,20 +227,16 @@ TEST_F(Daemon, proxy_contains_valid_info)
 
 TEST_F(Daemon, data_path_valid)
 {
-    QTemporaryDir data_dir, cache_dir;
-
-    EXPECT_CALL(mpt::MockStandardPaths::mock_instance(), writableLocation(mp::StandardPaths::CacheLocation))
-        .WillOnce(Return(cache_dir.path()));
+    QTemporaryDir storage_dir;
 
-    EXPECT_CALL(mpt::MockStandardPaths::mock_instance(), writableLocation(mp::StandardPaths::AppDataLocation))
-        .WillOnce(Return(data_dir.path()));
+    EXPECT_CALL(*mock_platform, multipass_storage_location()).WillOnce(Return(storage_dir.path()));
```

Comment:
You still want to test these if storage location comes back empty, that's where this coverage drop comes from, isn't it?

https://app.codecov.io/gh/canonical/multipass/compare/2485/changes#D1L117

Reply:
Ok, yeah, I need a new test case for this condition.

Reply:
Ok, really putting it back to (nearly) what it was is better.